<launch>

    <!-- Launch dynamic reconfigure -->
    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_console" args="-t"/>

    <node name="publish_node" pkg="template_package" type="publisher_node" output="screen"/>

    <node name="subscribe_node" pkg="template_package" type="subscriber_node" output="screen"/>	


</launch>